src/queries/__tests__/text.test.tsx

```diff
@@ -473,3 +473,15 @@ test('getByText and queryByText work properly with multiple nested fragments', (
   expect(getByText('Hello')).toBeTruthy();
   expect(queryByText('Hello')).not.toBeNull();
 });
+
+test('getByText and queryByText work with newlines', () => {
+  const { getByText, queryByText } = render(<Text>{'Line 1\nLine 2'}</Text>);
+  expect(getByText('Line 1\nLine 2')).toBeTruthy();
+  expect(queryByText('Line 1\nLine 2')).not.toBeNull();
```

Comment:
```suggestion
  expect(queryByText('Line 1\nLine 2')).toBeTruthy();
```

Reply:
I was following the existing test code in the file:
https://github.com/callstack/react-native-testing-library/pull/984/files#diff-c98a36aa080f209baf1bdb04f2012f180f654c992c0d5c79dc69e21775ec15abR474

I have changed my code only now.